crates/mun_hir/src/builtin_type.rs

```diff
@@ -53,13 +54,15 @@ impl BuiltinType {
         (name![i16],   BuiltinType::Int(BuiltinInt::I16)),
         (name![i32],   BuiltinType::Int(BuiltinInt::I32)),
         (name![i64],   BuiltinType::Int(BuiltinInt::I64)),
+        (name![i128], BuiltinType::Int(BuiltinInt::I128)),
```

Comment:
Add one more space here please. :)

Reply:
Yeah, I accidentally left a closing parenthesis after the `name!` part and forgot to add the space back.